fabfile.py

```diff
@@ -92,7 +92,10 @@ def decrypt_secrets(c):
 def install_deps(c: Connection):
     print('-- Installing dependencies')
     with c.cd(c.release_path):
-        c.run('source .venv/bin/activate && pipenv install --deploy', echo=True, env={'PIPENV_VENV_IN_PROJECT': 'true'})
+        # TODO: Edit out pipenv dependency
+        c.run("make venv")
+        c.run("source .venv/bin/activate")
```

Comment:
May want to double check this will actually activate the venv in the next call. IIRC Fabfile will actually separate each .run() call into a different SSH session, that's why I used && in the original.

That, or bake the venv sourcing into the makefile.

Reply:
Not sure how to bake the venv sourcing into the MakeFile -- I tried the naive just adding `source .venv/bin/activate` under `make venv`, but `make: source: Command not found`. Added all of them on the same line.

Reply:
Ah, this'll be because commands in the Makefile are run in a separate shell with `/bin/sh`: https://stackoverflow.com/a/43566158

Reply:
Hit 'em with the good old "bash -c "command".